README.md

```diff
@@ -6,6 +6,9 @@
 This project is a study case of some spring and kotlin libs
 
 # Getting Started
+
+To check if your application is up, use the health check route: `GET` [`http://localhost:8080/health`](http://localhost:8080/health)
+
```

Comment:
In my opinion we can put this information lately, because ours principal objective is have more than one project here, so the getting start need to inform how to call the docker-compose, to up all the applications evolved

Do you agree?